GraphicEditor/Repositories/BuildingRepository.cs

```diff
@@ -26,10 +26,9 @@ private List<Building> GetBuildings(String query)
                 entity.SerialNumber = (string)sqlReader[0];
                 entity.Name = (string)sqlReader[1];
                 entity.Color = (string)sqlReader[2];
-                entity.Shape = (string)sqlReader[3];
-                entity.Row = (int)sqlReader[4];
-                entity.Column = (int)sqlReader[5];
-                entity.Style = (string)sqlReader[6];
+                entity.Row = (int)sqlReader[3];
+                entity.Column = (int)sqlReader[4];
+                entity.Style = (string)sqlReader[5];
```

Comment:
Except this part other changes were made by college Stefan. 
Name of the building is needed for search, you shouldn't delete this. If you need serial nbrs for buildings that's fine but name of the building is also needed.

Reply:
You could delete this pull and add this few lines on your other pull which is based on the same subject. I don't see the reason for this pr when other is still active.

Reply:
he can merge into develop so everything will be ok when you rebase